### PR TITLE
DIV-6631 Judge cost order decisions in 'Outcomes of decree nisi' tab

### PR DESCRIPTION
**Pull request by ravinderpal**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-6631

Changes based on https://github.com/hmcts/div-ccd-definitions/pull/769

### Change description ###
- Changed DN outcome tab display order
- Added Label at top and label + label and JugCostsDecision field at bottom

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseField/CaseField-object-to-costs-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  {
+    "LiveFrom": "11/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "JudgeCostsOrderAdditionalInfo",
+    "Label": "Additional info",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "12/03/2021",
```
Comment (ravinderpal) [inline definitions/divorce/json/CaseField/CaseField-object-to-costs-nonprod.json:46]: From here onwards

File: definitions/divorce/json/CaseTypeTab/CaseTypeTab-object-to-costs-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+[
+  {
```
Comment (ravinderpal) [inline definitions/divorce/json/CaseTypeTab/CaseTypeTab-object-to-costs-nonprod.json:2]: this

**Review by ravinderpal — COMMENTED** [review]
Here are the changes I made for this pr. The rest are coming from DIV-6630 (ticket not merged yet)

**Comment by ravinderpal** [comment]
#769 has been merged

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-object-to-costs-nonprod.json [hunk 3]
```diff
@@ -208,5 +208,75 @@
     "CaseFieldID": "JudgeCostsOrderAdditionalInfo",
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "11/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "labelLegalAdvisorDecision",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "CRU"
```
Comment (shaunganley) [inline definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-object-to-costs-nonprod.json:217]: labels should only have Read permissions

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json [hunk 4]
```diff
@@ -14683,70 +14683,70 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "labelDnClarificationResponse",
     "UserRole": "caseworker-divorce-courtadmin",
-    "CRUD": "CRU"
+    "CRUD": "R"
```
Comment (ravinderpal) [inline definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json:14686]: Fixing previous label permissions as labels should have only R permission

**Review by shaunganley — APPROVED** [review]

**Review by Dizrupto — APPROVED** [review]

**Pull request merged**